recipe/meta.yaml

```diff
@@ -79,17 +79,14 @@ outputs:
         - attrs
         - click >=7.0, <=8.0.4
         - colorama
-        - dataclasses  # [py<37]
         - filelock
         - frozenlist
-        # We override the 1.43.0 upper-bound in setup.py here,
-        # see https://github.com/ray-project/ray/issues/28354
-        - grpcio >=1.32.0,<1.48
+        - grpcio >=1.32.0
         - jsonschema
         - msgpack-python >=1.0.0, <2.0.0
         - numpy >=1.20
         - pickle5  # [py<38]
-        - protobuf >=3.15.3, <4.0.0
+        - protobuf >=3.15.3
         - psutil
         - pyyaml
         - redis-py >=3.5.0  # [win]
```

Comment:
Maybe my setup.py foo is not strong enough, but I don't see a dependency on redis for windows. I am pretty sure the upstream package removed the redis requirement and now it is optional. See [this comment](https://github.com/ray-project/ray/issues/29323#issuecomment-1308285511), where I am trying to push one patch upstream.

Reply:
Thanks for catching that one. I'm picking some of those cleanups from #80 (that predate 2.1.0 and even 2.0.0) into #81